pages/common/xargs.md

```diff
@@ -2,7 +2,7 @@
 
 > Execute a command with piped arguments coming from another command, a file, etc.
 > The input is treated as a single block of text and split into separate pieces on spaces, tabs, newlines and end-of-file.
-> More information: <https://pubs.opengroup.org/onlinepubs/9699919799/utilities/xargs.html>.
+> More information: <https://man7.org/linux/man-pages/man1/xargs.1p.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/xargs.1pl>.
```

Reply:
Indeed, as per #5143 when referencing man pages we default to manned.org unless of course the man7.org version is better in some way.